thonny/plugins/micropython/uf2dialog.py

```diff
@@ -216,15 +216,15 @@ def _present_variants_for_target(self, target: TargetInfo) -> None:
             if variant.get("popular", False)
         }
         if populars and len(populars) < len(filtered_mapping):
-            enhanced_mapping = {"--- MOST POPULAR " + "-" * 100: {}}
+            enhanced_mapping = {"--- " + tr('MOST POPULAR ').ljust(100, '-'): {}}
             for variant in populars.values():
                 popular_variant = variant.copy()
                 # need different title to distinguish it from the same item in ALL VARIANTS
                 popular_title = self._create_variant_description(variant) + " "
                 popular_variant["title"] = popular_title
                 enhanced_mapping[popular_title] = popular_variant
 
-            enhanced_mapping["--- ALL VARIANTS " + "-" * 100] = {}
+            enhanced_mapping["--- " + tr('ALL VARIANTS ').ljust(100, '-')] = {}
```

Comment:
better move the trailing space outside of the `tr` argument

Reply:
You are right. I was experimenting a bit to make it working. And I think, my decision was based on some strings, which also ends with trailing space. I'll fix it.